dbt/adapters/duckdb/plugins/excel.py

```diff
@@ -0,0 +1,18 @@
+import pathlib
+from typing import Dict
+
+import pandas as pd
+
+from . import Plugin
+from ..utils import SourceConfig
+
+
+class ExcelPlugin(Plugin):
+    def __init__(self, config: Dict):
+        self._config = config
+
+    def load(self, source_config: SourceConfig):
+        ext_location = source_config.meta["external_location"]
+        ext_location = ext_location.format(**source_config.as_dict())
```

Comment:
This allows dynamically inserting values from the config using a format string. Could you give an example of how this is intended to be used? Does this, for example, contain the seed directory?